README.md

```diff
@@ -1,54 +1,30 @@
 [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![Build status](https://ci.appveyor.com/api/projects/status/brsd5poowi0moq2b/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/bhom-engine/branch/master)
+[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/BHoM_Engine/BHoM_Engine.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=15&branchName=master)
+# Core BHoM_Engine Repo
```

Comment:
For consistency with the BHoM ReadMe - can we either have a blank line above this header, or remove the blank line above the header in the BHoM ReadMe

Reply:
will add the extra line. does not change rendered layout but does help readability in raw markdown.